views/default/elgg.mjs

```diff
@@ -0,0 +1,7 @@
+const elgg = window.elgg;
+
+if (!elgg || !elgg._complete) {
+  throw new Error('Module cannot be used until elgg.js executes');
+}
+
+export default elgg;
```

Comment:
Note that unlike `define()`, we can't simply do async stuff before the module returns (only newish Chrome supports top-level await in ES modules). So we can't trivially use AMD's `require()` and export the AMD module value.

If `elgg.js` were rewritten as an ES module that would alleviate the need for a separate HTTP module just to make it available. An ES module can define window globals, but not the other way around.